src/apps/frontend/pages/authentication/signup/signup-form.tsx

```diff
@@ -36,6 +38,15 @@ const SignupForm: React.FC<SignupFormProps> = ({
   layoutType = LayoutType.Default,
 }) => {
   const { formik, isSignupLoading } = useSignupForm({ onSuccess, onError });
+  const currentLoginMethod = Config.getConfigValue<string>(
```

Comment:
The best thing to do here is not even mount the `/signup` route if auth type is mobile.